cypress/support/api/invoice.js

```diff
@@ -0,0 +1,19 @@
+Cypress.Commands.add('getInvoiceIdApi', (searchParams) => {
+  return cy
+    .okapiRequest({
+      path: 'invoice/invoices',
+      searchParams,
+    })
+    .then(responce => {
+      cy.log(JSON.stringify(responce));
```

Comment:
remove logging and pause from the result code

Reply:
Done